src/makeRobot.js

```diff
@@ -38,7 +38,61 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return ('name: ' + this.name + ', chip version: '
+      + this.version + ', wheels: ' + this.wheels);
```

Comment:
Use template string

```suggestion
      return (`name: ${this.name}, chip version: ${this.version},`
        + `wheels: ${this.wheels}`);
```
or
```suggestion
			const { name, version, wheels} = this;

      return (`name: ${name}, chip version: ${version}, wheels: ${wheels}`);
```